app/code/Codifi/CustomerRequest/Controller/Request/Save.php

```diff
@@ -0,0 +1,187 @@
+use Magento\Framework\Controller\ResultInterface;
+use Magento\Framework\Exception\NoSuchEntityException;
+use Magento\Framework\Exception\MailException;
+use Magento\Framework\App\Area;
+use Magento\Store\Model\Store;
+use Magento\Framework\Controller\ResultFactory;
+use Codifi\CustomerRequest\Helper\Config;
+use Magento\Framework\App\RequestInterface;
+
+/**
+ * Class Save
+ * @package Codifi\CustomerRequest\Controller\Request
+ */
+class Save extends Action
+{
+    /**
+     * Customer note factory
+     *
+     * @var CustomerNoteFactory
+     */
+    private $customerNoteFactory;
+
+    /**
+     * Customer note repository
+     *
+     * @var NoteRepository
+     */
+    private $customerNoteRepository;
+
+    /**
+     * Transport builder
+     *
+     * @var TransportBuilder
+     */
+    private $transportBuilder;
+
+    /**
+     * Store manager interface
+     *
+     * @var StoreManagerInterface
+     */
+    private $storeManager;
+
+    /**
+     * Config
+     *
+     * @var Config
+     */
+    private $config;
+
+    /**
+     * Save constructor.
+     *
+     * @param Context $context
+     * @param CustomerNoteFactory $customerNoteFactory
+     * @param NoteRepository $customerNoteRepository
+     * @param TransportBuilder $transportBuilder
+     * @param StoreManagerInterface $storeManager
+     * @param Config $config
+     */
+    public function __construct(
+        Context $context,
+        CustomerNoteFactory $customerNoteFactory,
+        NoteRepository $customerNoteRepository,
+        TransportBuilder $transportBuilder,
+        StoreManagerInterface $storeManager,
+        Config $config
+    ) {
+        $this->customerNoteFactory = $customerNoteFactory;
+        $this->customerNoteRepository = $customerNoteRepository;
+        $this->transportBuilder = $transportBuilder;
+        $this->storeManager = $storeManager;
+        $this->config = $config;
+        parent::__construct($context);
+    }
+
+    /**
+     * Execute function
+     *
+     * @return ResultInterface
+     * @throws LocalizedException
+     * @throws MailException
+     * @throws NoSuchEntityException
+     */
+    public function execute(): ResultInterface
+    {
+        $request = $this->getRequest();
+
+        $validate = $this->validate($request);
+
+        $customerId = $validate['customerId'];
+        $customerEmail = $validate['customerEmail'];
+        $message = $validate['message'];
+        $customerName = $validate['customerName'];
+
+        try {
+            $store = $this->storeManager->getStore();
+            $storeName = $store->getFrontendName();
+
+            $format = 'Customer sent request from %s';
+            $note = sprintf($format, $storeName);
+
+            $sender = [
+                'name' => $customerName,
+                'email' => $customerEmail
+            ];
+
+            $transportTemplate = $this->transportBuilder->setTemplateIdentifier('customer_request_template');
```

Comment:
Setters never return value. In this case, setTemplateIdentifier returns instance of TransportBuilder class.